lib/apipie/core_ext/route.rb

```diff
@@ -0,0 +1,3 @@
+ActionDispatch::Journey::Route.class_eval do
+  attr_accessor :base_url
```

Comment:
If we really need to extend some class we don't own, I would prefer using a module to add the additional code, so that it's easier to track where it's coming from.

Reply:
Feedback addressed!